src/Microsoft.Build.Sql/templates/Microsoft.Build.Sql.Templates.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <id>Microsoft.Build.Sql.Templates</id>
+        <!-- Version is set by build pipelines -->
+        <version>1.0.0</version>
+        <title>Microsoft SQL Database Project Build SDK templates</title>
+        <description>This package contains the templates for building SQL Database Projects (.sqlproj) in .NET Core and .NET 5+.</description>
+        <packageType>Template</packageType>
+        <authors>Microsoft</authors>
+        <owners>Microsoft</owners>
+        <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+        <projectUrl>https://github.com/microsoft/DacFx</projectUrl>
+        <repository type="git" url="https://github.com/microsoft/DacFx" branch="main" />
+        <tags>dacfx sqlproj build sdk</tags>
+    </metadata>
+    <files>
+        <file src="sqlproject\**" target="content" />
```

Comment:
We might not need a nuspec. You can include the files by adding this to the csproj
```xml
<ItemGroup>
  <Content Include"sqlproject/**" PackagePath="content/" />
</ItemGroup>
```